website/docs/using-unleash/deploy/upgrading-unleash.md

```diff
@@ -5,6 +5,52 @@ import VideoContent from '@site/src/components/VideoContent.jsx'
 
 Generally, the intention is that `unleash-server` should always provide support for clients one major version lower than the current one. This should make it possible to upgrade `unleash` gradually.
 
+## Upgrading to v6 from v5
+
+[Unleash v6](https://github.com/Unleash/unleash/issues/4380) was released on June 6th, 2024. **We expect this upgrade to be straightforward**, provided you followed the previous migration guides in this page. We're removing some features that were deprecated in version 5, new deprecated features that will continue to be supported during v6. Here's the list of the outstanding changes you need to take into account when migrating from v5 to v6:
+
+### Remove passport libs from the official Open Source Docker distribution
+
+*If you're not using the official open source image, you can safely ignore this change.* 
+
+The [official open source docker image](https://hub.docker.com/r/unleashorg/unleash-server) stop including these libraries which are custom authentication. If you're using this feature, there is a [community image](https://github.com/Unleash/unleash-docker-community) which adds these dependencies. 
+
+### Drop support for postgres versions 10, 11 and 12
+
+*If you're using postgres 13 or above you can safely ignore this change*
+
+Unleash v6 will output an error message when starting with an unsuported version but will continue to run. We recommend upgrading to a supported version as soon as possible.
+
+### Update to Node.js version 20+
+
+Unleash v6 drops support Node.js versions below 20, which is the [current active LTS at the time of release](https://github.com/nodejs/Release/blob/6209d04302e62156b964a605f619283582334c95/README.md#release-schedule).
+
+### Remove /edge/metrics endpoint
+
+*If you're not using edge, you can safely ignore this change*
+
+If you're using edge to connect to Unleash, make sure to first upgrade unleash-edge to version 19.1.3 or above.
+
+### Remove legacy "/api/feature
+
+If you're still using this endpoint, checkout the [deprecation notice in v4](#4-legacy-v2-routes-removed)
```

Comment:
```suggestion
If you're still using this endpoint, check out the [deprecation notice in v4](#4-legacy-v2-routes-removed)
```

Reply:
Also, what do they need to do if they are using it? It's not really clear from that paragraph you link to 😅  Maybe we don't have a clear path, and that's okay too, but I think it'd be nice if it was clearer what to do.